### PR TITLE
Updated IDsupplement.json

### PR DESCRIPTION
**Pull request by tekarrawilkinson**
Adding IDsupplement.json.

Review: https://brms-simulator-dev.apps.silver.devops.gov.bc.ca/rule/66c76fd01aeab403143c84b4?version=inReview

### CODE REVIEW

**Comment by rachel0728** [comment]
Ran -50.00 $ scenario and got an eligible result; we will need to update this as per camp fees.

**Comment by tekarrawilkinson** [comment]
> Ran -50.00 $ scenario and got an eligible result; we will need to update this as per camp fees.

rule has been rewritten to handle negative amounts @rachel0728

**Comment by rachel0728** [comment]
@tekarrawilkinson  I have validated that a -50.00 entry is working as expected with an eligible decision and 0.00 Thank you !

**Review by prchristenson — APPROVED** [review]
Validation Review completed.  Two scenarios added by IM and passed.  As policy/reg does not contain a specific $ limitation for the  supplement beyond "actual cost of the identification" no limit will be included in the rule set.    There will be a need to create utility rule that manages expense authority levels as that is a requirement that cross multiple supplements.  see [ADO user story 4135](https://dev.azure.com/bc-icm/Business-Rules/_sprints/backlog/Business-Rules%20Team/Business-Rules/Sprint%2011?workitem=3633).

**Review by Iliana777 — APPROVED** [review]
Validation review completed. @sbuhovac @rachel0728 ready for isolation testing.

File: rules/general-supplements/IDsupplement.json
Comment (brysonjbest) [inline rules/general-supplements/IDsupplement.json:1]: Removed extra 'new line' from the rule which was causing a rule run failure in csv format. CSV tests should be functional now.

**Review by sbuhovac — APPROVED** [review]
Isolation testing completed with all tests passed. CSV file attached. 
[general-supplements_IDsupplement_testing_general-supplements_IDsupplement_2024-09-19T23-35-43Z.csv](https://github.com/user-attachments/files/17067919/general-supplements_IDsupplement_testing_general-supplements_IDsupplement_2024-09-19T23-35-43Z.csv)

**Comment by rachel0728** [comment]
> Isolation testing completed with all tests passed. CSV file attached. [general-supplements_IDsupplement_testing_general-supplements_IDsupplement_2024-09-19T23-35-43Z.csv](https://github.com/user-attachments/files/17067919/general-supplements_IDsupplement_testing_general-supplements_IDsupplement_2024-09-19T23-35-43Z.csv)

I have reviewed and agree; all tests passed.

**Pull request merged**